sdk/python/tests/integration/online_store/test_python_feature_server.py

```diff
@@ -137,5 +135,5 @@ def python_fs_client(environment, universal_data_sources, request):
     feast_objects.extend([driver(), customer(), location()])
     fs.apply(feast_objects)
     fs.materialize(environment.start_date, environment.end_date)
-    client = TestClient(get_app(fs))
-    yield client
+    with TestClient(get_app(fs)) as client:
+        yield client
```

Comment:
[using test client as a context manager allows fastapi to incorporate lifespan](https://fastapi.tiangolo.com/advanced/testing-events/). dynamo needs this bc the connection and by extention event loop management happens at startup/shutdown. this fixed the remaining integration test issues.